src/locales/es_ES/es_ES.json

```diff
@@ -1,4 +1,91 @@
 {
+    "auth_accessibility": {
+        "back_button": "Cancelar"
+    },
+    "auth_cam_encrypt": {
+        "loader": "Cifrado de transmisión de la cámara"
+    },
+    "auth_cam_start": {
+        "loader": "Iniciando cámara"
+    },
+    "auth_capture_start": {
+        "body": "Coloque su rostro dentro del marco ovalado",
+        "button_primary": "Iniciar escaneo del rostro",
+        "feedback": {
+            "center_face": "Coloque su rostro en el marco",
+            "conditions_too_bright": "Busque un entorno con menos luz",
+            "conditions_too_dark": "Busque un entorno con más luz",
+            "head_not_upright": "Mantenga la cabeza erguida",
+            "neutral_expression": "Mantenga una expresión neutra",
+            "not_looking_straight": "Mire hacia adelante",
+            "remove_sunglasses": "Quítese las gafas de sol",
+            "steady_count_1": "No se mueva: 1",
+            "steady_count_2": "No se mueva: 2",
+            "steady_count_3": "No se mueva: 3"
+        },
+        "title": "Iniciar escaneo del rostro"
+    },
+    "auth_capture": {
+        "feedback": {
+            "center_face": "Coloque su rostro en el marco",
+            "even_lighting": "Asegúrese de que la iluminación sea uniforme",
+            "eye_level": "Mantenga la cámara a la altura de los ojos",
+            "face_not_found": "Asegúrese que su cara sea visible",
+            "head_not_upright": "Mantenga la cabeza erguida",
+            "move_back": "Ahora muévase hacia atrás",
+            "move_close": "Ahora acérquese",
+            "move_closer": "Acérquese",
+            "not_looking_straight": "Mire hacia adelante",
+            "steady": "No se mueva"
+        }
+    },
+    "auth_error": {
+        "cam_encryption": {
+            "body": "Esta aplicación bloquea las configuraciones sospechosas de las cámaras web. <a href='https:\/\/livenesscheckhelp.com\/' target='_blank' style='text-decoration:underline;'>Más información<\/a>.",
```

Comment:
This could leave us open to XSS attacks. You need to use an XML tag here and the `parseTags` utils function. Take a look at the CountrySelector component to get an idea of how it's done or message me or @kopijunkie. It needs to be changed also on Lokalise

Reply:
Are there any specific keys I should add the `parseTags` to? Or all of them?

Reply:
This would also cause any UI styling/customisation to not get applied since there isn't any class name for this link element.

Reply:
I'm confused. When you say link element what are you referring to? And where would a class name be needed for this case?

Reply:
This part `<a href='https:\/\/livenesscheckhelp.com\/' target='_blank' style='text-decoration:underline;'>`. It's usually handled by `parseTags` & needs a `<fallback>` tag configured on Lokalise side also. Let's pair on this. Will be easier than trying to explain in words. 😄

Reply:
Btw, you can call it whatever you want not necessarily `<fallback>`. You can call it <cameraHelpLink> or whatever makes sense to you in this scenario